Dockerfile

```diff
@@ -62,6 +62,20 @@ COPY config config
 COPY devtools devtools
```

Comment:
There's also a dockerfile in tools (Dockerfile.CI I think). If now, this is the ground truth for CI, let's delete it and update the README in tools/

Reply:
Even the README there is stale. Let me take care of it in a separate PR. There is also the pending work of deploying green builds to `live.arcs.dev`.